pkg/pillar/cas/containerd_test.go

```diff
@@ -0,0 +1,43 @@
+package cas
```

Comment:
This file is missing Copyright and license lines.

Reply:
Done